mip_convert/CHANGES.md

```diff
@@ -1,6 +1,6 @@
-## Release 3.1.2rc1, April 11 2025
+## Release 3.1.2, April 30 2025
 
--   No changes
+-   The `iris` `cell_methods` interval value is now ignored (CDDSO-648)
```

Comment:
```suggestion
-  `interval` specifications in the `cell_methods` field of input netCDF files are now ignored to avoid mid-year changes in model timestep causing iris to raise concatenation errors (CDDSO-648)
```